### DISCUSSION TITLE
Need additional smiley implying "OK"

### DISCUSSION THREAD

**Discussion by omkar3141** [community/community#43187] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Currently github has following kinds of smilies to interact with pull request comments.
![image](https://user-images.githubusercontent.com/31892352/210334674-3bc2f816-affc-4db1-bb6f-73e70ba252ad.png)

It will be really great if it can provide an "OK" smiley, to indicate acknowledgment from the reader of the comment. 

Example:
![image](https://user-images.githubusercontent.com/31892352/210335521-2da1d19f-eace-4e0f-ae81-830abbac1764.png)

Thanks!

**Comment by radar-blip** [comment]
🆗

**Reply by omkar3141** [reply]
This requires typing the symbol in the text `:ok`. That's unnecessary. Having it as a part of the emoticon set for 'responding' to comments, makes this action nimble and easy :). Github can probably do a survey of all public PR comments that match one or more of these words ("ok, fine, alright, that's correct, will do, done," etc) and you will see that all of these comments make comment thread little longer, occupies more screen area and are bit inefficient IMO.

**Comment by abwtrf089** [comment]
ÃÂÃÂ­ÃÂµÃÂÃÂ ÃÂ¹ÃÂ BlueMail for Mobile
David wrote:

**Comment by abwtrf089** [comment]
Ã™Â„Ã˜Â­Ã˜ÂµÃ™ÂˆÃ™Â„ Ã˜Â¹Ã™Â„ BlueMail for Mobile<https://bluemail.me/>
David wrote:



🆗

**Comment by jfullstackdev** [comment]
is the thumbs up not an `okay` for you ?

**Reply by omkar3141** [reply]
The tooltip for the "thumbs" up says, "+1". Through usage on some other social media platforms, the meaning of "👍"  has become multifaceted and can lead to wrong interpretation if used outside of the context provided by the tooltip. Ideally, there should have been a separate "+1" smiley/emoticon to indicate support for a given comment from co-reviewers, but we can't expect this to change because even the text shorthand reads `:+1`.

Hence the request for a separate smiley to indicate acknowledgment of the comment by the code-writer.

**Comment by spenserblack** [comment]
I kind of feel like :eyes: covers this. It can mean "I'm watching this" and also "I've seen this" IMO.

**Comment by abwtrf089** [comment]
احصل على Outlook لـ iOS<https://aka.ms/o0ukef>
